src/main/java/core/basesyntax/file/ReportWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.file;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ReportWriterImpl implements ReportWriter {
+    @Override
+    public void write(String data, String filePath) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(filePath))) {
+            writer.write(data); // Записуємо весь рядок у файл
```

Comment:
never use languages other than English in your code, you will most probably work with developers from abroad at some point